src/App.tsx

```diff
@@ -15,49 +17,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
```

Comment:
nit: Consider replacing 'ReorderOptions' with a more meaningful name like 'SortOptions' to match the sorting context 💡